src/GitVersion.Core/VersionCalculation/BaseVersionCalculators/VersionStrategyModule.cs

```diff
@@ -6,9 +6,9 @@ public class VersionStrategyModule : GitVersionModule
 {
     public override void RegisterTypes(IServiceCollection services)
     {
-        var versionStrategies = FindAllDerivedTypes<IVersionStrategy>(Assembly.GetAssembly(GetType())).Where(x => x != typeof(VersionStrategyBase));
+        var versionStrategies = FindAllDerivedTypes<IVersionStrategy>(Assembly.GetAssembly(GetType()));
 
-        foreach (var versionStrategy in versionStrategies)
+        foreach (var versionStrategy in versionStrategies.Where(x => !x.IsAbstract && !x.IsInterface))
```

Comment:
Why move the `.Where` clause to the `foreach`? Please just change the predicate in the one that existed on line 9.

Reply:
done